controllers/cronset_controller_test.go

```diff
@@ -60,4 +64,32 @@ var _ = Describe("CronSet controller", func() {
 			Expect(k8sClient.Create(ctx, cronJob)).Should(Succeed())
 		})
 	})
+
+	Context("Node Event", func() {
+		It("Should support watching for node events", func() {
+			ctx := context.Background()
+
+			By("Creating a new node")
+			node := &corev1.Node{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "node1",
+				},
+				Spec: corev1.NodeSpec{
+					ProviderID: "grasse:///test/1",
+				},
+			}
+			Expect(k8sClient.Create(ctx, node)).Should(Succeed())
+
+			By("Checking that the mapping function has been called")
+			nodeLookupKey := types.NamespacedName{Name: node.Name}
+			createdNode := &corev1.Node{}
+			Eventually(func() bool {
+				err := k8sClient.Get(ctx, nodeLookupKey, createdNode)
+				if err != nil {
+					return false
+				}
+				return true
+			}).Should(BeTrue())
+		})
+	})
```

Comment:
Did you just want to see that creating and getting the node using api thorugh this test?

Reply:
I wanted to see if the mapping function I added to `handler.EnqueueRequestsFromMapFunc` was called correctly.

Later, when the `Reconcile` method was changed to create a cronjob resource.. I'll be able to complete this test more fully.